Sources/SwiftPlot/HistogramSeries.swift

```diff
@@ -1,5 +1,7 @@
 public struct HistogramSeries<T> where T: Comparable {
-    public var data = [T]()
+    public var data = [T]() { didSet { isSorted = false } }
```

Comment:
If we want to protect a little against ourselves would it be a good idea to do what you suggest and also leaving the didSet?
My thinking is that, if we ever were to add more mutating methods to `HistogramSeries` in the future, and by chance forget to use the correct setter (lets call it `setData(_: [T], sorted: Bool)`, and instead we set `data` directly without setting `isSorted` to a correct value as well, it could lead to very subtle bugs where `isSorted` should be false after such a method but because it stayed true now the algorithm throws garbage.
If we would keep the `didSet`, even with a private(set), at least in these cases it wouldn't lead to mysterious bugs in some rare cases, but instead lead to a worse performance. I can see though why the better outcome in this case could be to generate a garbage graph, instead of a good one that was generated slower, in terms of how quickly the underlying bug would be caught.